db_store/src/lib.rs

```diff
@@ -12,6 +14,8 @@ pub enum MetaError {
     SqlError(#[from] sqlx::Error),
     #[error("Failed to decode meta value")]
     DecodeError,
+    #[error("not found")]
```

Comment:
```suggestion
    #[error("not found {0}")]
```
to report the missing string

Reply:
Hopefully fixed correctly in 4343973